dashboard/app/mailers/ops_mailer.rb

```diff
@@ -1,4 +1,4 @@
-require_relative '../../../lib/cdo/activity_constants'
+require_relative '../../../lib/cdo/workshop_constants'
```

Comment:
this should already be required by the require in workshop.rb I think. Does it work without this?